pkg/chartutil/create.go

```diff
@@ -147,16 +147,15 @@ service:
 
 ingress:
   enabled: false
+  className:
```

Comment:
would it be an idea to # this?
it's optional (because of ingress default class)

do you prefer to keep the annotation hint for kubernetes.io/ingress.class anyway?

Reply:
> would it be an idea to # this?

It has the same effect, but I think it's better this way.

> do you prefer to keep the annotation hint for kubernetes.io/ingress.class anyway?

That's an interesting question. I don't mind keeping it.

Reply:
when comparing to serviceaccount -> name, this might be another approach for the sake of consistency:  
same effect - but includes note for the customer.

```
# If not set, the default ingress class is used
className: ""
```

Reply:
Updated the default to string. Thanks!